src/externalServices/catalog/interfaces.ts

```diff
@@ -13,3 +13,8 @@ export interface Record3D extends Layer3DMetadata {
   id: string;
   links: string;
 }
+
+export interface DeleteRequest {
```

Comment:
Why did you add it here?

Reply:
I should have put it in the store triggre inrterface?

Reply:
Who is using this interface?

Reply:
the request call of store trigger
model manager and  test helpers